src/pages/FirstRun.vue

```diff
@@ -37,6 +37,11 @@
         :container-engine="settings.kubernetes.containerEngine"
         @change="onChangeEngine"
       />
+      <Checkbox
+        label="Enable Kubernetes"
+        :value="settings.kubernetes.enabled"
+        @input="handleDisableKubernetesCheckbox"
+      />
```

Comment:
The positioning of this seems odd (it should probably be at the _top_ of the section that gets disabled), but we can maybe do a cleanup pass afterwards.

Reply:
This will need a hopefully quick UI pass by Philip